todolist/settings.py

```
@@ -63,11 +63,11 @@
 DATABASES = {
     'default': {
         'ENGINE': 'mysql.connector.django',
-        'NAME': 'app_db',
-        'USER': 'app_user',
-        'PASSWORD': '1234',
-        'HOST': '172.17.0.2',  # You can use a different host if your MySQL server is on a remote machine.
-        'PORT': '',  # Leave this empty to use the default MySQL port (3306).
+        'NAME': os.environ.get('DB_NAME'),
```

Reading DB name from environment is correct. Consider providing a sensible default or explicit error message if `DB_NAME` is missing so the app fails fast with a clear message. For example: `os.environ.get('DB_NAME', 'app_db')` (and ensure this default matches docker-compose).

Also double-check that `requirements.txt` contains the appropriate driver (`mysql-connector-python`) because `ENGINE` is set to `mysql.connector.django`.